gtda/utils/validation.py

```diff
@@ -77,17 +77,36 @@ def check_graph(X):
 
 # Check the type and range of numerical parameters
 def validate_params(parameters, references):
+    """Function to automatize hyperparameters validation.
```

Comment:
Replace "automatize" with "automate"

Reply:
Thanks.

This function along with all the other one is `validation.py` is meant to be used by contributors. The documentation should be cristal clear. Your question above probably means that it is not yet the case. Please let me know.